LEM_GEOMETRY.s

```diff
@@ -189,7 +189,7 @@ RRNB1		PUSH	COS		# SHAFT ANGLE TO 2
 # THIS ENTRY TO RRNB REQUIRES THE TRUNNION AND SHAFT ANGLES IN MPAC AND MPAC +1 RESPECTIVELY
 
 RRNBMPAC	STODL	20D		# SAVE SHAFT CDU IN 21.
-			MPAC		# SET MODE TO DP.  (THE PRECEEDING STORE
+			MPAC		# SET MODE TO DP.  (THE PRECEDING STORE
```

Comment:
`PRECEEDING` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0324.jpg)